docs/API/Instance Methods.md

```diff
@@ -484,12 +548,16 @@ Observes the data at a particular [keypath](../Concepts/Templates/Keypaths.md).
 
 **Arguments**
 
-- `keypath (String)`: The [keypath](../Concepts/Templates/Keypaths.md) to observe, or a group of space-separated keypaths. Any of the keys can be a `*` character, which is treated as a wildcard.
+- `keypath (String)`: The [keypath](../Concepts/Templates/Keypaths.md) to observe, or a group of space-separated keypaths. Any of the keys can be a `*` character, which is treated as a wildcard. A doubled `**` wildcard means `recursive`.  
+
+    The difference between `*` and `**` is that `**` provides your callback function `value` and `keypath` arguments containing the path of the what actually changed, at any level of the keypath. So instead of getting the same parent value on every change, you get the changed value from whatever arbitrarily deep keypath changed.
 - `callback (Function)`: The function that will be called, with `newValue`, `oldValue` and `keypath` as arguments (see [Observers](../Concepts/Events/Publish-Subscribe.md) for more nuance regarding these arguments), whenever the observed [keypath](../Concepts/Templates/Keypaths.md) changes value. By default the function will be called with `ractive` as `this`. Any wildcards in the [keypath](../Concepts/Templates/Keypaths.md) will have their matches passed to the callback at the end of the arguments list as well.
 - `map (Object)`: A map of keypath-observer pairs.
 - `[options] (Object)`:
     - `[init] (boolean)`: Defaults to `true`. Whether or not to initialise the observer, i.e. call the function with the current value of `keypath` as the first argument and `undefined` as the second.
     - `[defer] (boolean)`: Defaults to `false`, in which case [observers](../Concepts/Events/Publish-Subscribe.md) will fire before any DOM changes take place. If `true`, the [observer](../Concepts/Events/Publish-Subscribe.md) will fire once the DOM has been updated.
+    - `[links] (boolean)`: Defaults to `false`.  Whether or not the observer should "follow through" any links created with [ractive.link()](#link).
+    - `[strict] (boolean)`: Defaults to `false`. {{{TODO need explanation of this option}}}
```

Comment:
You can use the same branch you used for this PR. Make the edits, commit the changes, and push the branch up. Github will automatically add the commit in this PR.